experiment/run-library.sh

```diff
@@ -167,9 +168,17 @@ function status_data_create() {
         sd_file="$sd_log.json"
         rm -f "$sd_file"
     fi
+    if [ -n "${RDD_FILE}" -a -f "${RDD_FILE}" ]; then
+        rdd_file=${RDD_FILE}
+    else
+        rdd_file="${sd_log}.rdd.json"
+        rm -f "${rdd_file}"
+    fi
     if [ -n "$PARAM_inventory" ]; then
         sd_hostname="$( ansible $opts_adhoc --list-hosts satellite6 2>/dev/null | tail -n 1 | sed -e 's/^\s\+//' -e 's/\s\+$//' )"
     fi
+    workdir_url="${PARAM_workdir_url:-https://workdir-exporter.example.com/workspace}"
+    sd_link="${workdir_url}/${JOB_NAME:-NA}/${sd_log}"
```

Comment:
Once the test finishes, we move the files to `.../$sd_run/`, is that part of `$sd_log`?